tools/debian/rules

```diff
@@ -11,6 +11,9 @@ endif
 %:
 	dh $@ --with=systemd
 
+clean:
+	dh $@
+
```

Comment:
Could you contribute a pull request to the release-dsc file in the cockpituous repository?